### DISCUSSION TITLE
can't send any email to check smtp in symfony 6

### DISCUSSION THREAD

**Discussion by Alfonsogreat** [symfony/symfony#45801] [Q&A] [answered]
hello everyone i have installed via composer the module "symfony/mailer" and also symfony/google-mailer to send email via gmail, but this is not working and i can see any email on my inboxes (PS: Less Secure app access = activated on Gmail) , and also to be sur that  is not a gmail problem i have used other smtp provider to test sending mails , but the email still not showing and the email boxes till empty, any help please ?

**Comment by glukose** [comment, marked as answer]
Howdy, Same problem with mailtrap on sf 6.0.6 
The debug toolbar does indicate one email Queued but 0 sent. 
I will try with other email providers to see if there is any difference.

(edit by @lyrixx - copy for another message) 
> I did found out that my problem was linked to the messenger and opened this issue : https://github.com/symfony/symfony/issues/45811

**Reply by glukose** [reply]
I voluntary changed my MAILER_DSN to `MAILER_DSN=smtp://+` but no error is raised. I believe there is an error when sending the email but nothing gets reported.

In the controller, I do try to catch any error but to no avail
```
try {
    $mailer->send($email);
} catch (TransportExceptionInterface $e) {
            
}
```

**Reply by lyrixx** [reply]
Have you messenger installed also? If it's the case, then you must run the consumer

**Reply by glukose** [reply]
@lyrixx Not sure what the consumer is.

I did found out that my problem was linked to the messenger and opened this issue : https://github.com/symfony/symfony/issues/45811

**Reply by lyrixx** [reply]
`bin/console messenger:consume async`

**Reply by jadhashas** [reply]
@glukose merci infiniment pour la solution , tu ma sauver <3

**Comment by tsizehena223** [comment]
Hi there, if anyone will look for this one day, then just comment this line : `Symfony\Component\Mailer\Messenger\SendEmailMessage: async` in your **messenger.yaml** in your `config/packages/messenger.yaml`

**Reply by ghost** [reply]
thanks, it worked

**Reply by MarlyJAM** [reply]
Merci beaucoup

**Reply by aflorczak** [reply]
thanks :)

**Reply by t5810m** [reply]
Thank you very much.

**Reply by zahitrios** [reply]
Thanks!!

**Reply by juricacenan** [reply]
Thanks for the info. It worked for me on my local environment. But when i push on production it does not work ? Any help is appreciated.

**Comment by land22** [comment]
thank you so much, you are the best !!!! everything work perfectly
